index.js

```diff
@@ -190,6 +192,18 @@ RingPop.prototype.setupChannel = function setupChannel() {
     createServer(this, this.channel);
 };
 
+RingPop.prototype.adminJoin = function adminJoin(callback) {
```

Comment:
Can you remove this from the RingPop prototype? We're trying to reduce the overall footprint of the "public" RingPop interface. These two shouldn't be here.